docs/notebooks/comparison_with_EventDisplay.ipynb

```diff
@@ -386,7 +388,10 @@
     "\n",
     "plt.errorbar(x, y, xerr=xerr, yerr=yerr, ls='', label=\"EventDisplay\")\n",
     "\n",
-    "for name in ('', '_NO_CUTS', '_ONLY_GH', '_ONLY_THETA'):\n",
+    "# uncomment the other strings to see effective areas\n",
+    "# for the different cut levels. Left out here for better\n",
+    "# visibility of the final effective areas.\n",
+    "for name in ('', ): #'_NO_CUTS', '_ONLY_GH', '_ONLY_THETA'):\n",
```

Comment:
maybe too picky as a comment, but as it is written, someone could do this
`for name in ('', ): '_NO_CUTS', '_ONLY_GH', '_ONLY_THETA'):`
which is wrong syntax.
Better put those keys in a list above the for cycle and comment/uncomment single 
entries